include/deal.II/lac/affine_constraints.templates.h

```diff
@@ -262,10 +262,12 @@ AffineConstraints<number>::add_entries(
                                          col_val_pair.first,
                                          entry.second,
                                          col_val_pair.second));
+            entry_exists = true;
             break;
           }
 
-      line.entries.push_back(col_val_pair);
+      if (entry_exists == false)
+        line_ptr->entries.push_back(*col_val_pair);
```

Comment:
should be `line.entries.push_back(col_val_pair);`.

Reply:
You are right. I fixed it first based on the source file from the release version 9.0, because I'm currently using this one. There col_val_pair is a std::vector<>::iterator.

Reply:
Only to be on the save side and doing the right things since this is the first time I'm running through this pull request process.
What I have to do know is to correct the 'push_back'-command argument and add a changelog file. After that I push the changes to the my remote branch again. That's all, isn't it?

Reply:
Yes, that would be the simplest way to move forward with your PR.